packages/calcite-components/src/components/button/button.e2e.ts

```diff
@@ -533,12 +533,12 @@ describe("calcite-button", () => {
         buttonEl = await page.find("calcite-button >>> button");
         await buttonEl.focus();
         await page.waitForChanges();
-        buttonFocusStyle = await buttonEl.getComputedStyle(":focus");
+        buttonFocusStyle = await buttonEl.getComputedStyle();
```

Comment:
Nitpick: `buttonFocusStyle` can be changed to `buttonStyle` here to be more generic. Applies to similar use cases.